tools/travis/setup.sh

```diff
@@ -30,12 +30,14 @@ function retry() {
   fi
 }
 
-sudo gpasswd -a travis docker
+#sudo gpasswd -a travis docker
```

Comment:
@csantanapr, can this line be deleted, or is it still needed?

Reply:
not sure if it's need it or note, wanted to try without it first

Reply:
removed